benches/jit_compile.rs

```diff
@@ -12,26 +12,36 @@ extern crate test;
 use solana_rbpf::{
     elf::Executable,
     user_error::UserError,
-    vm::{Config, EbpfVm, SyscallRegistry, TestInstructionMeter},
+    verifier::{Verifier, VerifierError},
+    vm::{Config, EbpfVm, SyscallRegistry, TestInstructionMeter, VerifiedExecutable},
 };
 use std::{fs::File, io::Read};
 use test::Bencher;
 
+struct TautologyVerifier {}
```

Comment:
Why define it over and over again? Can't we move it to either src/verifier.rs or test_utils/src/lib.rs ?